app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/account/manage_your_details.scala.html

```diff
@@ -71,7 +71,7 @@ <h2 class="heading-medium">
   @cyaRow(
    messages("subscription.cyaLabel.contactName"),
    Html(details.contactName.value),
-   Some(cyaChange(messages("subscription.cyaChange.contactName"), "#"))
+   Some(cyaChange(messages("subscription.cyaChange.contactName"), name.routes.SubscribedChangeContactNameController.enterContactName().url))
```

Comment:
we'd only go to `SubscribedChangeContactNameController.enterContactName()` if the user registered with id. If they went through the register without id route we'd need to take them to a new controller which let them enter in an `IndividualName` - were you going to do that in a subsequent PR or something?

Reply:
:man_facepalming: missed that - will add that now

Reply:
added, but see other comments